composer.json

```diff
@@ -11,9 +11,30 @@
 			"homepage" : "https://members-plugin.com/"
 		}
 	],
+    "scripts": {
+        "strauss": [
+            "vendor/bin/strauss",
+            "vendor/bin/cspf-gt-i18n-replace vendor-prefixed members"
+        ]
+    },
 	"require"     : {
-		"composer/installers" : "^1.0",
```

Comment:
I see you removed this but the vendor-prefixed contains `composer/installer/` and should not.
That's because I think you ran `strauss` when before removing this dependency or anyways when the `vendor` directory was not clean.
Consider that if you don't specify which package you want Strauss to process, it'll process by default all those that are in the `require` section.

Also I see that the growth-tools version in `vendor-prefixed` is not at the latest, plus each file contains the modified date (2023), and your new strauss configuration `"include_modified_date": false,`.

I suggest then to do the following:
- remove the `vendor` dir
- remove the `vendor-prefixed` dir
- remove the `composer.lock` file
- run `composer install`
- run `git add` (should add `composer.lock` and `vendor-prefixed` with the new growth-tools only)
- commit and push

Reply:
oh I see @allycspf already noted that here: https://github.com/caseproof/members/pull/131#pullrequestreview-2485077458